extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -699,6 +703,14 @@ export class AmpA4A extends AMP.BaseElement {
           checkStillCurrent();
           this.adUrl_ = adUrl;
           this.handleLifecycleStage_('urlBuilt');
+          // If we should skip the XHR, we will instead request and render
+          // by simply writing a frame into the page using
+          // renderViaIframeGet
+          if (!this.isXhrAllowed()) {
```

Comment:
Should we move the adUrl existence check up?  Currently you would fall down the iframe_get path if its not present (which I assume is fine but wanted to mention it)